api/v1beta1/disruption_types.go

```diff
@@ -515,7 +515,6 @@ func DisruptionIsNotReinjectable(kind chaostypes.DisruptionKindName) bool {
 var NoSideEffectDisruptions = map[chaostypes.DisruptionKindName]struct{}{
 	chaostypes.DisruptionKindNodeFailure:      {},
 	chaostypes.DisruptionKindContainerFailure: {},
-	chaostypes.DisruptionKindCPUPressure:      {},
```

Comment:
We now move several processes to other CGroups, even if we put great care to kill them if anything is going wrong, we are not exempt from bug, hence we might have a side effect.